.gitignore

```diff
@@ -38,3 +38,4 @@ build
 *.exe
 *.out
 *.app
+firmware/data/config.json
```

Comment:
Are you adding this file to ignore, but it is a required file for running the T-Stick. I suggest making some modifications and creating a template file in case designers want to quickly clone and flash without changing any T-Stick info.

```suggestion
```